administration/live-migration.adoc

```diff
@@ -0,0 +1,138 @@
+# Live Migration
+
+Live migration is a process during which a running Virtual Machine Instance
+moves to another compute node while the guest workload continues to run and
+remain accessable.
+
+## Initiate live migration
```

Comment:
Does a human operator need to open some feature gate in order to use LM?

Reply:
Yep, I've missed it. Added.